app/components/on-site-notifications/notification-card.js

```diff
@@ -295,6 +303,101 @@ export default Component.extend(
 			});
 		},
 
+		getMessageWallPostBody(model) {
+			let wallOwner = model.get('metadata') && model.get('metadata.username');
+		
+			if (!wallOwner) {
+				wallOwner = this.getMessageWallOwner(model.get('uri'));
+			}
+		
+			const isOwnWall = wallOwner === this.usernameMarkup;
+			const args = {
+				postTitle: this.postTitleMarkup,
+				wallOwner,
+			};
+		
+			if (isOwnWall) {
+				// "{user} left a <b>new message</b> on your wall <br><br> {postTitle}",
+				args.user = this.getPossiblyAnonActorName(model);
+				return this.getTranslatedMessage('notifications-own-wall-post', args);
+			}
+		
+			args.firstUser = this.getPossiblyAnonActorName(model);
+			// "{firstUser} left a <b>new message</b> on {wallOwner}'s wall <br><br> {postTitle}"
+			return this.getTranslatedMessage('notifications-wall-post', args);
+		},
+
+		getMessageWallReplyBody(model) {
+		let wallOwner = model.get('metadata') && model.get('metadata.username');
```

Comment:
Indentation is off for this whole function block.

Reply:
Updated.